scripts/generate_cli_api.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""REANA client CLI API docs generation."""
+
+import click
+
+from reana_client.cli import cli
+
+
+def _print_code_block(content):
+    print('```')
+    print(content)
+    print('```')
+
+
+print('# reana-client CLI API')
+with click.Context(cli) as ctx:
+    _print_code_block(cli.get_help(ctx))
+print('## Commands')
+for cmd_obj in cli.commands.values():
+    print('## {}'.format(cmd_obj.name))
+    print(cmd_obj.help)
```

Comment:
What do you think about encapsulating all of this inside a function which we can name, and where we can write a docstring explaining what it does? then we can call it right after.